<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Poppins;
	font-size: 16px;
}

QWidget {
	background-color: white;
}

QPushButton {
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}

QLabel {
	text-align: center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="initBtn">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>220</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #266676;
	color: white;
	font-size: 18px;
	font-weight: 500;
	height: 50px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
	color: #266676;
	font-size: 18px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Atrás</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tdsCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 10px;
	font: 16px &quot;Poppins&quot;;
	color: white;
}

QCheckBox::indicator {
    width: 20px; /* Ancho del indicador */
    height: 20px; /* Alto del indicador */
    border: 2px solid white; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: white; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Conductividad Eléctrica/TDS                                     </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Escoja los parámetros que desea monitorear&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
background-color: #e76f51;
color: white;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;La temperatura está activada por defecto&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>41</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22577a;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>76</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1a759f;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="phCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>76</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 10px;
	font: 16px &quot;Poppins&quot;;
	color: white;
}

QCheckBox::indicator {
    width: 20px; /* Ancho del indicador */
    height: 20px; /* Alto del indicador */
    border: 2px solid white; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: white; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>pH                                                                                         </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>111</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22577a;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="oxygenCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>111</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 10px;
	font: 16px &quot;Poppins&quot;;
	color: white;
}

QCheckBox::indicator {
    width: 20px; /* Ancho del indicador */
    height: 20px; /* Alto del indicador */
    border: 2px solid white; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: white; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Oxígeno Disuelto                                                             </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>146</y>
      <width>480</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1a759f;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="allCheckBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>146</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 10px;
	font: 16px &quot;Poppins&quot;;
	color: white;
}

QCheckBox::indicator {
    width: 20px; /* Ancho del indicador */
    height: 20px; /* Alto del indicador */
    border: 2px solid white; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: white; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Seleccionar todos                                                           </string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>initBtn</zorder>
   <zorder>backBtn</zorder>
   <zorder>tdsCheckBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>phCheckBox</zorder>
   <zorder>oxygenCheckBox</zorder>
   <zorder>allCheckBox</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
